EIPS/eip-2569.md

```diff
@@ -1,17 +1,15 @@
 ---
 eip: 2569
 title: Saving and Displaying Image Onchain for Universal Tokens
-author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous)
+author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous), Ran Xing (@lemontreeran)
+description: A set of interfaces to save an SVG image in Ethereum, and to retrieve the image file from Ethereum for universal tokens. 
```

Comment:
Please swap these two lines (description comes first).

Reply:
Hi @axic , Change is made according to your suggestion. Thanks a lot for your approval.